data/replaceOldMajors.js

```diff
@@ -0,0 +1,80 @@
+import { connect } from "./db.js";
+import User from '../model/User.js';
+import Plan from '../model/Plan.js';
+
+async function replaceOldWithNew() {
+  connect();
+
+  const users = await User.find({});
+  for (let user of users) {
+    for (let plan_id of user.plan_ids) {
+      let plan = await Plan.findById(plan_id).exec();
+      if (plan !== null && plan.majors != null && plan.majors.length > 0) {
+        // console.log("before: " + plan.majors);
+        for (let ind in plan.majors) {
+          let major = plan.majors[ind];
+          if (major.includes("B.A. Computer Science (NEW - 2021 & after)")) {
+            plan.majors[ind] = "B.A. Computer Science";
+          } else if (major.includes("B.S. Computer Science (OLD - Pre-2021)") 
+          || major.includes("B.S. Computer Science (NEW - 2021 & after)")) {
+            plan.majors[ind] = "B.S. Computer Science";
+          } else if (major.includes("Minor Computer Science (OLD - Pre-2021)")) {
+            plan.majors[ind] = "Minor Computer Science";
+          } else if (major.includes("Minor Computer Science (NEW - 2021 & after)")) {
+            plan.majors[ind] = "Minor Computer Science";
+          } else if (major.includes("Minor Applied Mathematics & Statistics (OLD - Pre-2021)")
+          || major.includes("Minor Applied Mathematics & Statistics (NEW - 2021 & after)")) {
+            plan.majors[ind] = "Minor Applied Mathematics & Statistics";
+          } 
+        }
+        plan.majors =  [...new Set(plan.majors)];
+        // console.log("after: " + plan.majors + "");
+        await plan.save();
+        // console.log("done!\n");
+      }
+      if (plan !== null && plan.major_ids != null && plan.major_ids.length > 0) {
+        console.log("before: " + plan.major_ids);
```

Comment:
when is it major_ids vs majors again? 

i think it's major_ids for DIS branch or something

Reply:
i do not know... As per the model typing, Plan should only have majors field. But as least for the mock users some of their plans have major_ids (some have both majors and major_ids).